lib/streamlit/web/server/server.py

```diff
@@ -119,11 +119,21 @@
 SCRIPT_RUN_CHECK_TIMEOUT = 60
 
 
+class SessionClientDisconnectedError(Exception):
+    """Raised by operations on a disconnected SessionClient."""
+
+    pass
```

Comment:
nit: I think the `pass` can be dropped here as the docstring itself counts as a nonempty line